crl/validator.go

```diff
@@ -143,7 +148,7 @@ func (v *validator) verifyCRL(crl *pkix.CertificateList) error {
 	return certificate.CheckCRLSignature(crl)
```

Comment:
Not sure if Nuts is already using Go 1.19 but `Certificate.CheckCRLSignature` is deprecated:

> Deprecated: Use RevocationList.CheckSignatureFrom instead.

Reply:
Yeah, I noticed. More work :'-)